wagn/lib/wagn/generators/wagn/templates/config/routes.erb

```diff
@@ -1,8 +1,6 @@
 # -*- encoding : utf-8 -*-
 
-require 'decko/engine'
-
-Rails.application.routes.draw do
+Wagn.application.routes.draw do
```

Comment:
This is wrong.  Now this will only be used for the engine deck cases, and we don't want any wagn dependencies there.  The actual Rails.application will have a class determined by the application, and it will not be Wagn related.  Does this even work?  I'm not sure Wagn.application actually exists, nor should it.